gatherers/service-worker.js

```diff
@@ -20,11 +20,15 @@ const Gather = require('./gather');
 
 class ServiceWorker extends Gather {
   setup(options) {
+    this.resolved = false;
+
     this.artifactsResolved = new Promise((res, _) => {
       options.driver.on(
```

Comment:
Not sure how I missed this, but in general we've been explicitly dropping driver out to its own variable. I'm definitely being pedantic here.